model/project_ref_test.go

```diff
@@ -1641,6 +1641,80 @@ func TestCreateNewRepoRef(t *testing.T) {
 	assert.NotContains(t, scope.Resources, doc1.Id)
 }
 
+func TestGitHubDynamicTokenPermissions(t *testing.T) {
+	perms := GitHubDynamicTokenPermissions{
+		GitHubDynamicTokenPermission{
+			Requesters: []string{"requester1", "requester2"},
+			Permissions: map[string]string{
+				"checks": "write",
+				"issues": "read",
+			},
+		},
+		GitHubDynamicTokenPermission{
+			Requesters: []string{"requester3"},
+			Permissions: map[string]string{
+				"content": "read",
+				"fake":    "write",
+			},
+		},
+	}
+
+	t.Run("Invalid requester specified", func(t *testing.T) {
```

Comment:
since this "requester0" is not necessarily an invalid requester, it's just not defined, can we please change the wording here?